lib/vehicle_factory.rb

```diff
@@ -0,0 +1,14 @@
+class VehicleFactory 
+
+  def create_vehicles(vehicles)
```

Comment:
This looks good. Another way you could simplify this would be to move line 11 up to line 5 like so and 
```ruby
Vehicle.new({
  engine: :ev,
  make: vehicle[:make],
  model: vehicle[:model],
  vin: vehicle[:vin_1_10],
  year: vehicle[:model_year
})
```